src/data-visualisation/dashboard-layout/info-pages.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: article
+title: "Layout of a dashboard: Information"
+description: "Overview of the information pages to include in NHSBSA reports"
+status: DRAFT
+tags: [data-viz, data-viz-layout]
```

Comment:
Tags don't include `dv-db-layout` so the collection will be hidden. Double check these for both pages. 
Should it just be:
`tags: dv-db-layout`?